attribute-service-api/build.gradle.kts

```
@@ -15,7 +15,7 @@ val generateLocalGoGrpcFiles = false
 
 protobuf {
   protoc {
-    artifact = "com.google.protobuf:protoc:3.15.6"
+    artifact = "com.google.protobuf:protoc:3.21.1"
```

we should keep the versions same for the modules coming from the same group - https://github.com/hypertrace/attribute-service/pull/160/files#diff-fd6def69030cceecee0ba666b24df9af4eb511efc9c8b849c0bc168a1596b812R17
```suggestion
    artifact = "com.google.protobuf:protoc:3.21.12"
```

---

Done